Documentation/deploy-master.md

```diff
@@ -339,7 +454,22 @@ $ sudo systemctl enable kubelet
 Created symlink from /etc/systemd/system/multi-user.target.wants/kubelet.service to /etc/systemd/system/kubelet.service.
 ```
 
-### Create kube-system Namespace
+### Start Calico
+
+Now start Calico (if required)
```

Comment:
can we give you a hint about the test to know if its required or not?

"Start Calico if you configured your kubelet to use CNI plugins earlier:"